app/assets/javascripts/package.json

```diff
@@ -22,7 +22,7 @@
   ],
   "resolutions": {
     "**/babel-plugin-debug-macros": "npm:@discourse/babel-plugin-debug-macros@0.4.0-pre1",
-    "**/babel-plugin-ember-template-compilation": "2.0.0",
+    "**/jsesc": "npm:jsesc@3.0.2",
```

Comment:
What's the reason for this new pin?

Reply:
Good question! 😅 I don't remember the context, might have been unrelated change. Seem to be working with lower versions too - removing.